### ISSUE TITLE
[Bug Report] v-text-field in list lacks bottom border

### ISSUE THREAD

**Issue by rlipenta-medicity** [vuetifyjs/vuetify#3847]
### Versions and Environment
**Vuetify:** 1.0.14
**Vue:** 2.5.16
**Browsers:** Chrome 65.0.3325.181
**OS:** Windows 8.1

### Steps to reproduce
Creating a form that is similar to Google Contacts Add Contact form embedded within a dialog. Using a list to enable optional icons to the left, and optional actions to the right or each form element. View reproduction link example to see dialog and form.

### Expected Behavior
Expect the v-text-field to include its underline border when contained within a list embedded in a dialog.

### Actual Behavior
Lacks the underline border

### Reproduction Link
[https://codepen.io/anon/pen/ZxNggR](https://codepen.io/anon/pen/ZxNggR)

### Other comments
Also, alignment of the form element is shown below the action icon and button. This is worked around using "pt-3" on the list actions.

**Comment by johnleider** [comment]
This is resolved in v1.1

**Comment by greatwillow** [comment]
At the moment with the latest version of Vuetify v1.1 alpha 5, I am also getting a related UI issue.  I can create a separate post for this if this is placed in the incorrect ticket.  The following screenshot should describe the issue.  

In this case, I am using:

**Vuetify** v1.1 alpha 5
**Vue** v2.4.4
**chrome** v66.0.3359.139
**macOS** 10.13

![screen shot 2018-05-16 at 2 58 41 pm](https://user-images.githubusercontent.com/15651001/40138078-e34137ce-5919-11e8-9647-b2a9f58ed4c9.png)

**Comment by johnleider** [comment]
I don't understand. The bottom lines are there.

**Comment by rlipenta-medicity** [comment]
The issue was occurring with 1.0.14, but also occurs in 1.0.18. I see it no longer happens with 1.1.0-alpha-5, however the width of the field is now different.

Original issue - missing field underline/bottom line:
![image](https://user-images.githubusercontent.com/13201219/40142000-76935cea-5925-11e8-838c-2e2bb1945bd3.png)

With 1.1.0-alpha.5:
![image](https://user-images.githubusercontent.com/13201219/40142045-96b37cbc-5925-11e8-9adb-5251f1cb892f.png)

**Comment by greatwillow** [comment]
Yes, for all of them except the phone number field.  I have tried different variations and different fields in different positions in such a list will go without a bottom border.  I have figured out a workaround to force a bottom border with css selectors, though probably not the desired outcome.

**Comment by johnleider** [comment]
If someone is able to provide a reproduction using an alpha codepen I'll reopen and investigate.

**Comment by greatwillow** [comment]
Actually I've discovered this may not be necessary.  I realized that the reason for the behaviour was because my browser window was not at 100% zoom.  The behaviour is as expected when the zoom level is 100%.  It is strange, however, that when zooming out, only some of the borders will disappear.  This may only be the case in Chrome, as I also checked in Safari and it seems that the behaviour is more consistent.  If anyone wants to check this out, it is possible to observe just by going to the  on the Vuetify forms demo page [here](https://vuetifyjs.com/en/components/forms) (with chrome) and zooming out.  Not sure if this is so much a Vuetify bug as a Chrome bug.  Any thoughts on this?
